Content.Shared/Store/Components/StoreComponent.cs

```diff
@@ -41,7 +42,7 @@ public sealed partial class StoreComponent : Component
     /// regardless of who activated it. I.E. role specific items for uplinks.
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite)]
-    public EntityUid? AccountOwner = null;
+    public Entity<MindComponent?>? AccountOwner = null;
```

Comment:
This looks like it should be a datafield and in turn an entityuid, I'm not really sure why it isn't a datafield.